tensorflow/python/compiler/tensorrt/test/trtengineop_input_output_shape_test.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Shall we rename this file to trt_engine_op_input_output_shape_test.py or trt_engine_op_shape_test.py?

Reply:
Fixed